on-boarding-modules/pytest-examples/conftest.py

```diff
@@ -2,6 +2,8 @@
 import os
 from selenium import webdriver
 
+# skipping the first file
+collect_ignore = ["test_module1_pytest.py"]
```

Comment:
Why ignore module1 test?

Reply:
Because that's the module that requires users to hard code `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`. It's basically the same test as module2